index.js

```diff
@@ -169,4 +185,52 @@ class CircuitBreaker {
     }
 }
 
-module.exports = CircuitBreaker;
+/* Class representing a collection of CircuitBreaker instances */
+class FuseBox {
+    /**
+     * Construct a FuseBox object
+     * @constructor
+     */
+    constructor() {
+        this.fuses = [];
+    }
+
+    /**
+     * Create a circuit breaker and add it to the fuse box
+     * @method   createFuse
+     * @param  {Function}   command                      Command to instantiate the circuit breaker with
+     * @param  {Object}     [options]                    Options to pass on to the circuit breaker
+     */
+    createFuse(command, options) {
+        const self = this;
+        const breaker = new CircuitBreaker(command, options);
+
+        breaker.on('open', () => self.tripFuses());
+        self.fuses.push(breaker);
+    }
+
+    /**
+     * Add an existing circuit breaker to the fuse box
+     * @method   addFuse
+     * @param  {CircuitBreaker}   breaker                The circuit breaker to be added to the fuse box
+     */
+    addFuse(breaker) {
+        const self = this;
+
+        breaker.on('open', () => self.tripFuses());
+        this.fuses.push(breaker);
+    }
+
+    /**
+     * Trip all the circuit breakers inside the fuse box.
+     * @method   tripFuses
+     */
+    tripFuses() {
+        this.fuses.map(fuse => fuse.forceOpen());
+    }
+}
+
+module.exports = {
```

Comment:
Please update the `README` to describe the new features as well as how to integrate with the old features (since the interface changed).